Update dateSerializationFormat (#6044)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCalendar/1 Configuration/dateSerializationFormat.md

---
---
##### shortDescription
Specifies the date-time value serialization format.

---

> Date-time serialization involves converting date-time objects into a string format for storage or transmission. To ensure proper format detection, specify this property.

Reviewer: I just want to use "conversion" instead of "converting". 

```suggestion
Date-time serialization involves date-time value conversion into a string format for storage or transmission. To ensure proper format detection, specify this property.
```

Use LDML patterns to pass [custom format strings](/Documentation/Guide/Common/Value_Formatting/#Format_UI_Component_Values/Custom_Format_String) to this property.

[important] **dateSerializationFormat** does not support all LDML pattern combinations.

For instance, you can specify the following patterns:

- `"yyyy-MM-dd"`    

> A local date.  

Reviewer: I wouldn't say "local date". I guess the word "local" should imply that you don't specify a particular time zone. But if you just specify a date (a Birth Date, or a Hire Date, or the Deadline Date) - I don't think anyone considers it to be a specific time that is different in different time zones. It's just "February 19th. 2024". 

```suggestion
A date.  
```

- `"yyyy-MM-ddTHH:mm:ss"`    
Local date and time.  

- `"yyyy-MM-ddTHH:mm:ssZ"`    

> The UTC date and time.  

Reviewer: ```suggestion
Date and time in UTC.  
```

- `"yyyy-MM-ddTHH:mm:ssx"`, `"yyyy-MM-ddTHH:mm:ssxx"`, `"yyyy-MM-ddTHH:mm:ssxxx"`    
Date and time with a timezone.

[note]

- You can use this property only if you do not specify the initial [value](/api-reference/10%20UI%20Components/dxCalendar/1%20Configuration/value.md '/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value'). **dateSerializationFormat** is calculated automatically if you pass a value in the initial configuration.

- If you specify this property, the **value** will be a string, not a Date object.

> - If use API to change the **value**, make sure that the value has the same format that you specified in this property.

Reviewer: ```suggestion
- If you use API to change the **value**, make sure that the value has the same format that you specified in this property.
```

> - This property applies only if the **forceIsoDateParsing** field is set to **true** in the [global configuration object](/api-reference/50%20Common/utils/config(config).md '/Documentation/ApiReference/Common/utils/#configconfig').

Reviewer: I feel like the verb "applies" is more of a direct russian-to-english translation. That verb probably requires an object... I would use the phrase "takes effect" instead. 

```suggestion
- This property takes effect only if the **forceIsoDateParsing** field is set to **true** in the [global configuration object](/api-reference/50%20Common/utils/config(config).md '/Documentation/ApiReference/Common/utils/#configconfig').
```
